### PR TITLE
Doxygen changes for Tiago

### PR DESCRIPTION
**Pull request by longhathuc**
- Add doxygen pages for Tiago

### CODE REVIEW

File: doc/main-page/01_xenial-install-simulation.hh [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+To have the simulation
+
+    sudo apt-get install robotpkg-tiago-simulation 
+
+The next step is to set your environment variables. You can automatize this by following the steps described here. Please read the instructions carefully. Setting bash variables might be tricky and lead to failure.
+
+<b>From source </b>
+
+The best way is to use robotpkg.
```
Comment (jmirabel) [inline doc/main-page/01_xenial-install-simulation.hh]: Is this still true or is it the remainder of a copy-paste ?

Comment (longhathuc) [reply doc/main-page/01_xenial-install-simulation.hh]: It is the remainder of copy-paste from wiki.

Comment (jmirabel) [reply doc/main-page/01_xenial-install-simulation.hh]: This means that we will have two documentations for the same thing and this is not good. I think it is better to put on the wiki what is already there. Would you mind updating the wiki instead for the files which were copy-pasted ?

Comment (longhathuc) [reply doc/main-page/01_xenial-install-simulation.hh]: Done, I have updated the instruction build from source both on git and the wiki files

Comment (jmirabel) [reply doc/main-page/01_xenial-install-simulation.hh]: Thanks.

After an oral discussion, we agreed on adding a note on the wiki that says the up-to-date doc is inside this package.

File: tests/stop-controllers.py [hunk 2]
```diff
@@ -1,10 +1,13 @@
...
 import rospy
 from controller_manager_msgs.srv import SwitchController
 
 rospy.init_node('StoppingControllers')
 swCtl = rospy.ServiceProxy('/controller_manager/switch_controller', SwitchController)
-resp = swCtl(stop_controllers=[
-    'mobile_base_controller', 'arm_controller', 'head_controller', 'torso_controller', 'gripper_controller'
-],
-             strictness=2)
+for ctrl in [ 'mobile_base_controller', 'arm_controller', 'head_controller', 'torso_controller', 'hand_controller', 'gripper_controller' ]:
```
Comment (jmirabel) [inline tests/stop-controllers.py:8]: I didn't check this script. Is the name of the controller `hand_controller` correct ?

Comment (longhathuc) [reply tests/stop-controllers.py:8]: yes, it's hand_controller

**Pull request merged**